lib/datadog/di/logging.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Datadog
+  module DI
+    # Logging for DI
+    module Logging
```

Comment:
Minor: Since every class that uses `Logging` ends up needing to get passed a `logger` object anyway, I wonder if it would make sense to turn this module into an actual class that would have a `logger` as an instance variable, and then you'd always call methods on this class instead of the logger directly.

This would also solve the potential (small) sharp edge of "oops I included Logging but this class doesn't already have a `logger`" -- since you know that if you have the object, then you have all that you need.